### DISCUSSION TITLE
Decouple Client From Server

### DISCUSSION THREAD

**Discussion by PizzaConsole** [oqtane/oqtane.framework#697] [General] [answered]
So I noticed that you can't deploy and client by itself like it talks about [here](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-3.1#standalone-deployment). I would love to be able to deploy the client app separately from the server code. Thoughts?

**Comment by mikecasas** [comment]
@PoisnFang I agree. Not sure what is the "pipleline"? It seems the initial releases have settled down, and we need to revisit the V.Next task list. Number 1 priority is docs. I think @iJungleboy is taking the lead on that. After that, we need to get a couple ideas to work on. I know @sbwalker is busy and he can't do it all alone. Maybe we can get together on Slack or a Zoom call once a month to discuss ideas. Just throwing some ideas out there.

**Reply by PizzaConsole** [reply]
I wouldn't be opposed to a monthly call and taking responsibility of a couple tasks. I am using this framework extensively.

**Reply by hishamco** [reply]
> Maybe we can get together on Slack or a Zoom call once a month to discuss ideas. Just throwing some ideas out there.

There's a Gitter room created also GitHub Discussion is a great feature, furthermore I requested @@sbwalker long time back to create a core team from an active member to take the responsibility, reduce the load, accelerate the process ... etc

**Reply by PizzaConsole** [reply]
> There's a Gitter room created

I am in the Gitter room. Though IMO something like Slack or Discord is much more useful and organized.

**Reply by iJungleboy** [reply]
@mikecasas I have the lead on the docs #570 but ATM I'm still waiting for 2-3 things from @sbwalker before I can actually start.

**Comment by PizzaConsole** [comment, marked as answer]
Closing due to discussion in #907
